app/src/main/java/br/org/cesar/discordtime/stickysessions/navigation/router/Router.java

```diff
@@ -0,0 +1,38 @@
+package br.org.cesar.discordtime.stickysessions.navigation.router;
+
+import android.app.Activity;
+
+import br.org.cesar.discordtime.stickysessions.R;
+import br.org.cesar.discordtime.stickysessions.navigation.exception.InvalidRouteException;
+import br.org.cesar.discordtime.stickysessions.navigation.repository.IRouteRepository;
+import br.org.cesar.discordtime.stickysessions.navigation.wrapper.IContextWrapper;
+
+public class Router implements IRouter {
+
+    private IContextWrapper mContext;
+    private IRouteRepository routeRepository;
```

Comment:
Follow Field Naming Conventions
e.g.:
- prefix **m** for member attributes;
- prefix **s** for member static attributes